src/RegionKit.csproj

```diff
@@ -16,6 +16,8 @@
 		<PackageReadmeFile>../README.md</PackageReadmeFile>
 		<PackageLicenseFile>../LICENSE</PackageLicenseFile>
 		<DebugType>Portable</DebugType>
+		<AppOutputBase>$(MSBuildProjectDirectory)\</AppOutputBase>
+    	<PathMap>$(AppOutputBase)=RegionKit\</PathMap>
```

Comment:
This might have also changed output folder, because the dll and the docstrings xml are in the diff now. Should probably delete those then gitignore the new location.

Reply:
oh, this might be a result of using the new versioning system (so the files are under `mod/newest/plugins` now)

i'll add it to the gitignore and delete it from the diff